### ISSUE TITLE
[SUGGESTION] Allow implementation for Android with TypeScript/JavaScript

### ISSUE THREAD

**Issue by Galiza** [Instabug/instabug-cordova#67]
Once we download the plugin and install all the dependencies, for us to start using it in both platforms, we have write the code in two different languages (TypeScript/JavaScript and Java), but it should be able to do it only through TypeScript/JavaScript.
For example the iOS implementation for Cordova:
```
cordova.plugins.instabug.activate(
    {
        ios: 'MY_IOS_TOKEN'
    },
    'shake',
    function () {
        console.log('Instabug initialized.');
    },
    function (error) {
        console.log('Instabug could not be initialized - ' + error);
    }
);
```
Like this, it'll be easier for newcomers and people with not a lot of experience in working with mobile apps.

**Comment by alyezz** [comment]
@Galiza Thanks for the suggestion! We're currently working on it.

**Comment by Galiza** [comment]
@alyezz For any luck of mine, would you have an estimate time about when will this update be coming out?

**Comment by alyezz** [comment]
@Galiza It's included in our latest release v8.6.5
